jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -219,15 +222,107 @@
             export KUBE_TEST_SCRIPT="./hack/jenkins/verify-dockerized.sh"
             ./hack/jenkins/gotest-dockerized.sh
+      - build-test-e2e-gce: # kubernetes-pull-build-test-e2e-gce
+          status-context: GCE e2e
+          success-comment: |
+            GCE e2e build/test **passed** for commit ${{ghprbActualCommit}}.
+            * [Build Log](http://pr-test.k8s.io/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/build-log.txt)
+            * [Test Artifacts](https://console.developers.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/artifacts/)
+            * [Internal Jenkins Results](${{JOB_URL}}/${{BUILD_NUMBER}})
+          failure-comment: |
+            GCE e2e build/test **failed** for commit ${{ghprbActualCommit}}.
+            * [Build Log](http://pr-test.k8s.io/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/build-log.txt)
+            * [Test Artifacts](https://console.developers.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/artifacts/)
+            * [Internal Jenkins Results](${{JOB_URL}}/${{BUILD_NUMBER}})
+          error-comment: |
+            GCE e2e build/test **errored** for commit ${{ghprbActualCommit}}.
+            * [Build Log](http://pr-test.k8s.io/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/build-log.txt)
+            * [Test Artifacts](https://console.developers.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/artifacts/)
+            * [Internal Jenkins Results](${{JOB_URL}}/${{BUILD_NUMBER}})
+          trigger-phrase: 'e2e'
+          cmd: |
+            if [[ -d "${{WORKSPACE}}/_output" ]]; then
+              sudo rm -rf "${{WORKSPACE}}/_output"
+            fi
+            export KUBE_SKIP_PUSH_GCS=y
+            export KUBE_RUN_FROM_OUTPUT=y
+            ./hack/jenkins/build.sh
+            if [[ "${{ghprbTargetBranch:-}}" == "master" || "${{ghprbTargetBranch:-}}" == "release-1.2" || "${{ghprbTargetBranch:-}}" == "release-1.3" ]]; then
+              # Nothing should want Jenkins $HOME
+              export HOME=${{WORKSPACE}}
+
+              export KUBERNETES_PROVIDER="gce"
+              export E2E_MIN_STARTUP_PODS="1"
+              export KUBE_GCE_ZONE="us-central1-f"
+              export FAIL_ON_GCP_RESOURCE_LEAK="true"
+
+              # XXX Not a unique project
+              export E2E_NAME="e2e-gce-${{NODE_NAME}}-${{EXECUTOR_NUMBER}}"
+              export GINKGO_PARALLEL="y"
+              # This list should match the list in kubernetes-e2e-gce.
+              export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
+              export FAIL_ON_GCP_RESOURCE_LEAK="false"
+              export PROJECT="kubernetes-jenkins-pull"
+              # Override GCE defaults
+              export NUM_NODES="6"
```

Comment:
ouch.

any thoughts on whether we should try to share some config between PR and post-submit e2e?

obviously don't want to change any behavior in this PR, but...